doc/user-guide/hierarchical-data.rst

```diff
@@ -644,3 +644,164 @@ We could use this feature to quickly calculate the electrical power in our signa
 
     power = currents * voltages
     power
+
+.. _alignment-and-coordinate-inheritance:
+
+Alignment and Coordinate Inheritance
+------------------------------------
+
+.. _data-alignment:
+
+Data Alignment
+~~~~~~~~~~~~~~
+
+The data in different datatree nodes are not totally independent. In particular dimensions (and indexes) in child nodes must be aligned (LINK HERE) with those in their parent nodes.
```

Comment:
This is where I would like to link to some generic documentation on what alignment is, but it doesn't really exist, see https://github.com/pydata/xarray/issues/9500.

Reply:
done in https://github.com/pydata/xarray/pull/9501/commits/b245bdd220e0bdfb75b8771b56dba5f0bdc4ce3e